src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const SUBWORD_MATCHER = /([A-Z]?[a-z]+|[A-Za-z]+)/g;
+export const SUBWORD_MATCHER = /[A-Z]?[a-z]+|[A-Z]+(?![a-z])|[0-9]+/g;
```

Comment:
@knausj85 shall we steal this regex for `create_spoken_forms`?

Reply:
Nice, I'll steal this.